chapter/related-work.md

```diff
@@ -8,9 +8,58 @@ Additionally, this part aims to recognize alternative approaches to the LSP, in
 
 ## Language Servers
 
+The LSP project was announced [@lsp-announced] in 2016 to establish a common protocol over which language tooling could communicate with editors.
+This way it is able to fully concentrate on source analysis instead of integration with specific editors by building custom GUI elements and being restricted to editors extension interface.
```

Comment:
Which way? what is "it"? I would suggest to be more explicit: "The LSP helps the language intelligence tooling to fully concentrate on source code analysis, independently of the user interface and the other peculiarities of each editor."